client/web/hilbert-heartbeat.js

```diff
@@ -43,7 +43,7 @@
                 debugLog("RESPONSE: " + xhttp.responseText);
             }
         };
-        var fullUrl = url + "/" + command + "?" + interval + "&appid=" + appId + "&cache_buster=" + new Date().getTime()
+        var fullUrl = url + "/" + command + "?" + (interval/1000.0) + "&appid=" + appId + "&cache_buster=" + new Date().getTime()
```

Comment:
Sure. That was only a temporary hotfix as there was no time to rebuild lots of images right before the final demo presentation. 

Will be consistently fixed within issue #6.